Cargo.toml

```diff
@@ -8,5 +8,6 @@ members = [
   "examples/geometry",
   "examples/rondpoint",
   "examples/sprites",
-  "examples/todolist"
+  "examples/todolist",
+  "regression_tests/i356-enum-without-int-helpers",
```

Comment:
I didn't want to land this without a test, and I didn't want to contrive a new example just to test this edge-case (especially since the bug depends on the *absence* of something in the UDL, rather than its presence). So, I'm trying out a new suite of explicit regression tests.

The setup here is a little wordy for my liking, and if we generate more of these then we may want to invest in some automation so they can be written more succinctly. But it's a start!

Reply:
I'm so pleased you have a test!

I think we could bike shed this, but happy to land as is.

FWIW, I think my pref would be:

`fixtures/regressions/enums-without-i32-helper` and then put the URL to the issue in the README.md of the fixture.

Reply:
WFM, I will update to the suggested layout. Something about the shorter directory names is very pleasing :-)